README.md

```diff
@@ -141,19 +156,26 @@ $ pytest tests/test_file_name.py
 
 ... where `test_file_name.py` is relpaced with the correct test file name.
 
-## Project Write-Up: How to Complete and Submit
+To run a single test by name:
+
+```bash
+# Must be in activated virtual environment
+$ pytest tests/test_file_name.py::test_name
+```
+
+... where `test_name.py` is relpaced with the name of the function.
+
+## Play Testing
 
-The goal of this project is to write code in `main.py` so that as many of the tests pass as possible.
+While we will mainly use a Test Driven Development (TDD) workflow for this project, it can be helpful to run code independently from running tests. To do this, a file `play_tester.py` is provided.
 
-To complete this project, use the above workflow and follow these steps:
+To run this file, use:
 
-1. Start with making the tests in `test_wave_01.py` pass.
-1. Review your code in `main.py` and see if there are ways you can make the code more readable.
-1. Then, work on making the tests in `test_wave_02.py` pass.
-1. Review your code in `main.py`
-1. Repeat on all test files until submission time.
+```bash
+python3 play_tester.py
+```
 
-At submission time, no matter where you are, submit the project via Learn.
+There is some starter code provided in `play_tester.py`. This code prints the test data that is used for many of the tests. Looking closely at this data can help us think critically about the expected output for given input for each function. Then, calling each function with this data allows us to observe the **actual** output for given input. 
```

Comment:
this is so helpful, esp since we cleaned up our test files and can no longer just check the shape of the input by glancing at the test